src/Collection.php

```diff
@@ -542,6 +586,55 @@ public function replaceDocument($documentId, array $content, array $options = []
     }
 
     /**
+     * Scrolls through specifications using the provided scrollId
+     *
+     * @param string $scrollId
+     * @param array $options Optional parameters
+     * @return mixed
+     * @throws \Exception
+     */
+    public function scrollSpecifications($scrollId, array $options = [])
+    {
+        if (!$scrollId) {
+            throw new \Exception('Collection.scrollSpecifications: scrollId is required');
+        }
+
+        $options['httpParams'] = [':scrollId' => $scrollId];
+
+        $data = [];
+
+        $response = $this->kuzzle->query(
+            $this->kuzzle->buildQueryArgs('collection', 'scrollSpecifications'),
+            $data,
+            $options
+        );
+
+        return $response['result'];
+    }
+
+    /**
+     * Searches specifications across indexes/collections according to the provided filters
+     *
+     * @param array $filters Optional filters in ElasticSearch Query DSL format
+     * @param array $options Optional parameters
+     * @return mixed
+     */
+    public function searchSpecifications(array $filters = [], array $options = [])
```

Comment:
@stafyniaksacha As discussed together, we chosed to use plain JSON instead of creating  aSpecification entity which would be overkill.